var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -165,25 +165,31 @@ def mpi_launcher(self):
         commands = ["srun", "mpirun", "mpiexec"]
         return which(*commands, path=searchpath) or which(*commands)
 
-    def test(self):
-        if self.spec.satisfies("@2020.10.00") or "+mpi" not in self.spec:
-            print("Skipping: stand-alone tests")
-            return
+    def test_slate(self):
+        """Test slate with +mpi and other options"""
```

Comment:
This isn't as descriptive as it could be, especially since only one of the examples is actually executed *and* there is a PR to deprecate `+mpi`.